ui/packages/atlasmap/src/Atlasmap/toolbarItems.tsx

```diff
@@ -15,17 +24,98 @@ import {
   TableIcon,
   ColumnsIcon,
   TrashIcon,
+  CaretDownIcon,
 } from "@patternfly/react-icons";
 import { css, StyleSheet } from "@patternfly/react-styles";
+import { useToggle } from "../UI";
 
 const styles = StyleSheet.create({
   toggled: { color: "var(--pf-global--primary-color--100) !important" },
 });
 
+export interface IAtlasmapToolbarItemProps {
+  showImportAtlasFileToolbarItem: boolean;
+  showImportJarFileToolbarItem: boolean;
+  showExportAtlasFileToolbarItem: boolean;
+  showResetToolbarItem: boolean;
+  onImportAtlasFile: (file: File) => void;
+  onImportJarFile: (file: File) => void;
+  onExportAtlasFile: () => void;
+  onResetAtlasmap: () => void;
+}
+export const AtlasmapToolbarItem: FunctionComponent<IAtlasmapToolbarItemProps> = ({
+  showImportAtlasFileToolbarItem,
+  showImportJarFileToolbarItem,
+  showExportAtlasFileToolbarItem,
+  showResetToolbarItem,
+  onImportAtlasFile,
+  onImportJarFile,
+  onExportAtlasFile,
+  onResetAtlasmap,
+}) => {
+  const { state: isOpen, toggle: onToggle, toggleOff } = useToggle(false);
+  const runAndClose = (cb: (...args: any[]) => any) => {
+    return (...args: any[]) => {
+      cb(...args);
+      toggleOff();
+    };
+  };
+  const dropdownItems = [
+    showImportAtlasFileToolbarItem && (
+      <ImportAtlasFileToolbarItem
+        onFile={runAndClose(onImportAtlasFile)}
+        key="import-catalog"
+      />
+    ),
+    showImportJarFileToolbarItem && (
+      <ImportJarFileToolbarItem
+        onFile={runAndClose(onImportJarFile)}
+        key="import-java-archive"
+      />
+    ),
+    (showImportAtlasFileToolbarItem || showImportJarFileToolbarItem) && (
+      <DropdownSeparator key="import-separator" />
+    ),
+    showExportAtlasFileToolbarItem && (
+      <ExportAtlasFileToolbarItem
+        onClick={runAndClose(onExportAtlasFile)}
+        key={"export-catalog"}
+      />
+    ),
+    showExportAtlasFileToolbarItem && (
+      <DropdownSeparator key="export-separator" />
+    ),
+    showResetToolbarItem && (
+      <ResetToolbarItem
+        onClick={runAndClose(onResetAtlasmap)}
+        key="reset-catalog"
+      />
+    ),
+  ].filter((f) => f);
+  return (
+    <ToolbarItem>
+      <Dropdown
+        toggle={
+          <DropdownToggle
+            id="atlasmap-toggle"
+            onToggle={onToggle}
+            iconComponent={CaretDownIcon}
+            data-testid="atlasmap-menu-button"
+          >
+            Atlasmap
```

Comment:
guessing this is the label for the import/export dropdown, AtlasMap (capital M) might be better for user facing label?

Reply:
would be cool to have just the new AM logo - just a thought...

Reply:
🤦‍♂️ every-single-time. Thanks! I need to write a plugin for eslint to catch these...

Reply:
> would be cool to have just the new AM logo - just a thought...

Makes sense, we can try that once ready.